README.md

```diff
@@ -122,7 +122,7 @@ L = Lx = Ly = Lz = 2000  # Length of each dimension.
 topology = (Periodic, Periodic, Bounded)
 
 model = NonhydrostaticModel(
-            grid = RectilinearGrid(CPU(); topology=topology, size=(Nx, Ny, Nz), extent=(Lx, Ly, Lz)),
+            grid = RectilinearGrid(CPU(); topology, size=(Nx, Ny, Nz), extent=(Lx, Ly, Lz)),
```

Comment:
I'm wondering if we should use a better example here, what do you think?

Reply:
I think it's all right... What would be simpler?

Reply:
I put in a two-dimensional turbulence example that isn't huge and actually does something instead.